src/public/meeting.ts

```diff
@@ -66,6 +66,17 @@ export namespace meeting {
     tenantId?: string;
   }
 
+  export interface LiveStreamState {
+    /**
+     * indicates whether meeting is streaming
+     */
+    isStreaming: boolean;
+    /**
+     * indicates whether meeting is streaming to your specific app
+     */
+    isStreamingByCurrentApp: boolean;
```

Comment:
We only want one property for now: `isStreaming`. Later we can add support for `isStreamingByOtherApp` or similar.

Reply:
fixed